Added note to install a package (#399)
redwoodjs/redwoodjs.com: docs/builds.md

yarn zip-it-and-ship-it dist/functions/ zipballs/
```

> Each lambda function in `./api/dist/functions` is parsed by zip-it-and-ship-it resulting in a zip file per lambda function that contains all the dependencies required for that lambda function.   
> Note: The `@netlify/zip-it-and-ship-it` package needs to be installed as a dev dependency in `api`.
> You can learn more about the package [here:] (https://www.npmjs.com/package/@netlify/zip-it-and-ship-it) . For more information on AWS Serverless Deploy see the [docs](https://redwoodjs.com/docs/deploy#aws-serverless-deploy) here.
> Also learn how to install in yarn (--dev or -D) as a dev dependency [here](https://classic.yarnpkg.com/en/docs/cli/add/#toc-yarn-add-dev-d).

Reviewer: ```suggestion
Each lambda function in `./api/dist/functions` is parsed by zip-it-and-ship-it resulting in a zip file per lambda function that contains all the dependencies required for that lambda function.
   
>Note: The `@netlify/zip-it-and-ship-it` package needs to be installed as a dev dependency in `api/`. Use the command `yarn workspace api add -D @netlify/zip-it-and-ship-it`.
>- You can learn more about the package [here](https://www.npmjs.com/package/@netlify/zip-it-and-ship-it). 
>- For more information on AWS Serverless Deploy see these [docs](https://redwoodjs.com/docs/deploy#aws-serverless-deploy).
```

I've added the installation command in the Note, used the "blockquote" formatting for the Notes section, added bullets and fixed link issue, and removed the information about installing as a dependency.

@lumenCodes you can accept this commit directly or make changes yourself. Also, once you've made a commit, you can preview the changes on the Website using the netlify/redwoodjs/deploy-preview link in the GitHub Checks.

<img width="573" alt="Screen Shot 2020-10-10 at 9 48 33 AM" src="https://user-images.githubusercontent.com/2951/95660585-c5f2b580-0add-11eb-92a9-8bd58894ceea.png">

Author: Thank you so much @thedavidprice I am really grateful. 
I think it is all fine to me :star_struck:
